drivers/hmis/spec/models/hmis/form/form_processor_spec.rb

```diff
@@ -409,6 +409,34 @@
       expect(disabilities.find_by(disability_type: 8).t_cell_count_available).to eq(1)
     end
 
+    it 'processes HOPWA fields (regression #6864)' do
+      assessment = Hmis::Hud::CustomAssessment.new_with_defaults(enrollment: e1, user: u1, form_definition: fd, assessment_date: Date.yesterday)
+      assessment.form_processor.hud_values = {
+        'DisabilityGroup.hivAids' => 'YES',
+        'DisabilityGroup.tCellCountAvailable' => 'YES',
+        'DisabilityGroup.tCellCount' => '222',
+        'DisabilityGroup.tCellSource' => 'MEDICAL_REPORT',
+        'DisabilityGroup.viralLoadAvailable' => 'AVAILABLE',
+        'DisabilityGroup.viralLoad' => '333',
+        'DisabilityGroup.viralLoadSource' => 'CLIENT_REPORT',
+        'Enrollment.disablingCondition' => 'YES',
```

Comment:
While working on this, I noticed that for hiv/aids, "indefinite and impairs" is collected as 99 rather than 1. I was a bit surprised by this, but based on the hud [data standards manual](https://files.hudexchange.info/resources/documents/HMIS-Data-Standards-Manual-2024.pdf) it appears to be correct behavior.